src/Tribe/RSVP/Service_Provider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Tribe\Tickets\RSVP;
+
+/**
+ * Class Service_Provider
+ *
+ * @since TBD
+ *
+ * @package Tribe\Tickets\RSVP
+ */
+class Service_Provider extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		$this->register_early_access();
+	}
+
+	/**
+	 * Registers the RSVP Early Access Service.
+	 *
+	 * @since TBD
+	 */
+	private function register_early_access() {
+		add_action( 'init', [
+			$this->container->make( Early_Access::class ),
+			'maybe_register_early_access_assets'
+		]);
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: 607f52e1958e8e661a8ed35000f44d60</sub>